Sources/HummingbirdHTTP2/HTTPServerBuilder+http2.swift

```diff
@@ -30,9 +30,10 @@ extension HTTPServerBuilder {
     ///   - tlsConfiguration: TLS configuration
     ///   - additionalChannelHandlers: Additional channel handlers to call before handling HTTP
```

Comment:
Can we document where these handlers are added?

Reply:
This is fixed in the HTTP2 PR